tests/test_base_models.py

```diff
@@ -528,3 +532,60 @@ class TestSequenceLabelerOscar(TestSequenceLabelerTextCNN):
 class TestClassifierOscar(TestClassifierTextCNN):
     model_specific_config = {"n_epochs": 2, "lr": 1e-4}
     base_model = OSCAR
+
```

Comment:
Think standard practice for this is to try / except these imports at the top, set an indicator variable, and then use skipIf()